packages/netlify-cms-widget-markdown/src/MarkdownControl/renderers.js

```diff
@@ -237,9 +237,13 @@ function NumberedList(props) {
 function Link(props) {
   const data = props.node.get('data');
   const url = data.get('url');
-  const title = data.get('title');
+
+  // title doesn't seem to return anything
```

Comment:
`title` returns something is you edit the markdown manually to `[text](url "title")`

Reply:
Oh, you're right, didn't think of that, thanks!